src/Sarif.UnitTests/Writers/SarifLoggerTests.cs

```diff
@@ -34,24 +33,38 @@ public void SarifLogger_RedactedCommandLine()
             // "C:\Program Files (x86\\Microsoft Visual Studio 14.0\Common7\IDE\QTAgent32_40.exe\" 
             // /agentKey a144e450-ac06-46d0-8365-c21ea7872d23 /hostProcessId 8024 /hostIpcPortName 
             // eqt -60284c64-6bc1-3ecc-fb5f-a484bb1a2475"
+            // 
```

Comment:
Having said that, this is an awful lot of trouble to test a simple string replacement routine. If your goal is to test the string replacement itself, you could just declare `SarifLogger.Redact` as `internal`. If your goal is to convince yourself that the `SarifLogger` ctor actually _calls_ `Redact`, then yeah, I guess you have to do it this way.